mifospay/src/main/java/org/mifos/mobilewallet/mifospay/qr/ui/ShowQrActivity.java

```diff
@@ -102,13 +102,12 @@ public void onClick(DialogInterface dialog, int which) {
                 generateQR(qrData + ", " + mAmount);
             }
         });
-        editTextDialog.setNeutralButton("Reset", new DialogInterface.OnClickListener() {
+        editTextDialog.setNegativeButton("Cancel", new DialogInterface.OnClickListener() {
```

Comment:
Please revert the hardcoded string form here. First, add the string in the `string.xml` file then use it dynamically.